modules/consensus/consensusset_test.go

```diff
@@ -87,6 +87,11 @@ func createConsensusSetTester(name string) (*consensusSetTester, error) {
 	return cst, nil
 }
 
+// closeCst safely closes the consensus set tester. 'close' is a builtin
+func (cst *consensusSetTester) closeCst() error {
+	return cst.gateway.Close()
+}
```

Comment:
@lukechampine 

Should instead this be `cst.cs.Close()` and then the consensus set calls `gateway.Close()`? Or do you think that modules should not close their dependencies.

I guess modules should not close their dependencies.